db/schema.rb

```diff
@@ -56,6 +56,7 @@
     t.datetime "updated_at", precision: 6, null: false
     t.bigint "payment_intent_id", null: false
     t.boolean "refunded", default: false
+    t.decimal "amount", null: false
```

Comment:
I don't think we want to do this since Amount is variable for GiftCards. We don't save the amount directly on the Item so that we can save a history. See here: https://docs.google.com/drawings/d/13ln9-dCYl_wUxLVkMfCH5VFOe9BgkzY0Qu8V-TTl0Fs/edit

Reply:
I always thought that GiftCardAmount was going to be used to track the balance of the gift card